collect_app/src/test/java/org/odk/collect/android/http/CollectServerClientTest.java

```diff
@@ -0,0 +1,42 @@
+package org.odk.collect.android.http;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.odk.collect.android.http.mock.MockHttpClientConnection;
+import org.odk.collect.android.utilities.DocumentFetchResult;
+
+@RunWith(JUnit4.class)
+public class CollectServerClientTest {
+
+    private TestableCollectServerClient collectServerClient;
+
+    @Before
+    public void setup() {
+        collectServerClient = new TestableCollectServerClient(new MockHttpClientConnection());
+    }
+
+    @Test
+    public void testGetXMLDocumentErrorResponse() {
+        collectServerClient.setGetHttpShouldReturnNull(true);
+        DocumentFetchResult fetchResult = collectServerClient.getXmlDocument("http://testurl");
+        Assert.assertEquals(fetchResult.errorMessage, "Parsing failed with null while accessing http://testurl");
```

Comment:
Would you consider statically importing these assert methods for better readability?